release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -405,6 +405,12 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="new-network-observability"]
+==== New Network Observability Operator to observe network traffic flow
+The cluster administrators can now observe the network traffic for {product-title} clusters by installing the Network Observability Operator. The Network Observability uses the eBPF technology to create network flows. The network flows are enriched with {product-title} information and stored in Loki. As an administrator, you can monitor the network traffic in the {product-title} console for detailed troubleshooting and analysis. This feature helps administrators to get insights from different graphical representations of traffic flow.
```

Comment:
Suggest: "A cluster administrator..."  
  
By using the word "clusters" in the first sentence, it seems to imply that Network Observability can manage more than one cluster.  Suggest "can now observe the network traffic for an {product-title} cluster by installing..."  
  
Suggest: "The Network Observability uses eBPF technology".  Remove "the".  
  
Suggest: "This feature helps admiinstrators get insights".  Remove "to".